AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react';
+import {View, Text, TouchableOpacity} from 'react-native';
+import {constants} from '../../../utils';
+import {ResultStyles} from '../../../styles';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import moment from 'moment'
+
+const DateDone = () => {
+  const [date, setDate] = useState(new Date(1598051730000));
```

Comment:
what this data is hardcoded?

Reply:
It was an example, and we forgot to modify the data